src/commands/transaction/reconstruct_transaction/mod.rs

```diff
@@ -0,0 +1,264 @@
+        let network_config = networks
+            .get(&scope.network_name)
+            .expect("Failed to get network config!")
+            .clone();
+        let query_view_transaction_status = network_config
+            .json_rpc_client()
+            .blocking_call(near_jsonrpc_client::methods::EXPERIMENTAL_tx_status::RpcTransactionStatusRequest {
+                transaction_info: near_jsonrpc_client::methods::EXPERIMENTAL_tx_status::TransactionInfo::TransactionId {
+                    hash: scope.transaction_hash.into(),
+                    account_id: "near".parse::<near_primitives::types::AccountId>()?
+
+                }
+            })
+            .wrap_err("Failed to fetch query for view transaction")?;
+
+        let signer_id = query_view_transaction_status
+            .final_outcome
+            .transaction
+            .signer_id;
+        let receiver_id = query_view_transaction_status
+            .final_outcome
+            .transaction
+            .receiver_id;
+        let archival_actions: Vec<Action> = query_view_transaction_status
+            .final_outcome
+            .transaction
+            .actions
+            .into_iter()
+            .map(near_primitives::transaction::Action::try_from)
+            .collect::<Result<_, _>>()
+            .expect("Internal error: can not convert the action_view to action.");
+        let prepopulated_transaction = crate::commands::PrepopulatedTransaction {
+            signer_id: signer_id.clone(),
+            receiver_id: receiver_id.clone(),
+            actions: archival_actions.clone(),
+        };
+
+        eprintln!("\nArchive transaction:\n");
+        crate::common::print_unsigned_transaction(&prepopulated_transaction);
+        eprintln!();
+
+        let cmd = crate::commands::CliTopLevelCommand::Transaction(
+            crate::commands::transaction::CliTransactionCommands{
+                transaction_actions: Some(crate::commands::transaction::CliTransactionActions::ConstructTransaction(
+                    crate::commands::transaction::construct_transaction::CliConstructTransaction {
+                        sender_account_id: Some(signer_id.into()),
+                        receiver_account_id: Some(receiver_id.into()),
+                        next_actions: None
+                    }
+                ))
+            }
+        );
+        let mut cmd_cli_args = cmd.to_cli_args();
+
+        for archival_action in archival_actions {
+            let next_actions = crate::commands::transaction::construct_transaction::add_action_1::CliNextAction::AddAction(
+                crate::commands::transaction::construct_transaction::add_action_1::add_action::CliAddAction{
+                    action: action_transformation(archival_action)?
+                }
+            );
+            cmd_cli_args.extend(next_actions.to_cli_args());
+        }
+
+        let skip_action = crate::commands::transaction::construct_transaction::add_action_2::CliNextAction::Skip(
```

Comment:
I believe that `_1` should work just fine:

```suggestion
        let skip_action = crate::commands::transaction::construct_transaction::add_action_1::CliNextAction::Skip(
```